runtime/tests/interpreter/account_test.go

```diff
@@ -205,6 +206,91 @@ func TestInterpretAuthAccount_save(t *testing.T) {
 	})
 }
 
+func TestInterpretAuthAccount_typeAt(t *testing.T) {
+
+	t.Parallel()
+
+	t.Run("typeAt", func(t *testing.T) {
+
+		t.Parallel()
+
+		address := interpreter.NewAddressValueFromBytes([]byte{42})
+
+		inter, accountStorables := testAccount(
+			t,
+			address,
+			true,
+			`
+              struct S {}
+
+              resource R {}
+
+              fun saveR() {
+				let r <- create R()
+				account.save(<-r, to: /storage/x)
+              }
+
+			  fun saveS() {
+				let s = S()
+				destroy account.load<@R>(from: /storage/x)
+			 	account.save(s, to: /storage/x)
+			  }
+
+              fun typeAt(): Type? {
+				return account.typeAt(/storage/x)
+              }
```

Comment:
The test case is a bit brittle here: The bug above where the result of the function wasn't wrapped in a `SomeValue` properly wasn't caught, because the non-optional value was automatically boxed due to the `Type?` return type. Maybe return `AnyStruct` instead and add a comment why